kmath-ejml/src/test/kotlin/space/kscience/kmath/ejml/EjmlMatrixTest.kt

```diff
@@ -22,65 +25,59 @@ fun <T : Any> assertMatrixEquals(expected: StructureND<T>, actual: StructureND<T
 internal class EjmlMatrixTest {
     private val random = Random(0)
 
-    private val randomMatrix: SimpleMatrix
+    private val randomMatrix: DMatrixRMaj
         get() {
             val s = random.nextInt(2, 100)
-            return SimpleMatrix.random_DDRM(s, s, 0.0, 10.0, random.asJavaRandom())
+            val d = DMatrixRMaj(s, s)
+            RandomMatrices_DDRM.fillUniform(d, random.asJavaRandom())
+            return d
         }
 
     @Test
     fun rowNum() {
         val m = randomMatrix
-        assertEquals(m.numRows(), EjmlMatrix(m).rowNum)
+        assertEquals(m.numRows, EjmlMatrixDouble(m).rowNum)
     }
 
     @Test
     fun colNum() {
         val m = randomMatrix
-        assertEquals(m.numCols(), EjmlMatrix(m).rowNum)
+        assertEquals(m.numCols, EjmlMatrixDouble(m).rowNum)
     }
 
     @Test
     fun shape() {
         val m = randomMatrix
-        val w = EjmlMatrix(m)
-        assertEquals(listOf(m.numRows(), m.numCols()), w.shape.toList())
+        val w = EjmlMatrixDouble(m)
```

Comment:
Why EjmlMatrixDouble and not EjmlDoubleMatrix?

Reply:
Maybe `DoubleEjmlMatrix` then?

Reply:
I like EjmlDoubleMatrix better since it would be easier to discover.

Reply:
OK